crates/noirc_frontend/src/hir/type_check/expr.rs

```diff
@@ -30,16 +30,13 @@ pub(crate) fn type_check_expression(
         HirExpression::Literal(literal) => {
             match literal {
                 HirLiteral::Array(arr) => {
-                    // Type check the contents of the array
```

Comment:
I thought the comments were too obvious and thus unhelpful. I like to include comments within functions only to explain why we're doing something rather than explaining what we're doing.